roles/sap_hana_install/README.md

```diff
@@ -2,7 +2,11 @@
 
 Ansible role for SAP HANA Installation
 
-## Prerequisites
+## Requirements
+
+The role requires additional collections which are specified in `meta/collection-requirements.yml`. These are not automatically installed.  You must install them like this:
```

Comment:
Suggest overly verbose statement.


This Ansible Role has a dependency requirement on other Ansible Collections, which are specified in `meta/collection-requirements.yml`. These are not automatically installed and are necessary before execution of this Ansible Role, such as installation of the requirements from Ansible Galaxy using:
`ansible-galaxy install -vv -r meta/collection-requirements.yml`

Reply:
Solved in https://github.com/sap-linuxlab/community.sap_install/commit/a1662546383075fe18f2303391e0fe7d80d82aaa .